automatic/tortoisesvn/tortoisesvn.nuspec

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>tortoisesvn</id>
+    <title>TortoiseSVN</title>
+    <version>1.9.5.27581</version>
```

Comment:
please change this to latest version available on chocolatey, or to 0.0. Otherwise it may not be updated.